src/app/portlets/content-types/main/content-types.component.spec.ts

```diff
@@ -27,32 +39,41 @@ describe('ContentTypesPortletComponent', () => {
 
         DOTTestBed.configureTestingModule({
             declarations: [
-                ActionHeaderComponent,
-                ActionButtonComponent,
                 ContentTypesPortletComponent,
-                ListingDataTableComponent
             ],
-            imports: [RouterTestingModule.withRoutes([
-                { path: 'test', component: ContentTypesPortletComponent }
-            ])],
+            imports: [
+                RouterTestingModule.withRoutes([
+                    { path: 'test', component: ContentTypesPortletComponent },
+                ]),
+                BrowserAnimationsModule,
```

Comment:
Double check this imports because some of them al already imported in other modules which means you don't need them here.

Reply:
It's needed in the specs, if I remove it this error will be displayed: 
"Error: Found the synthetic property @overlayState. Please include either "BrowserAnimationsModule" or "NoopAnimationsModule" in your application. ".

Reply:
I don't mean just BrowserAnimationsModule I mean double check all of them.

Reply:
Done, I just removed the unnecessary imports.